File: Python3/프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.py
def solution(s):
    char_count = {}
    for char in s:
        if char in char_count:
            char_count[char] += 1
        else:
            char_count[char] = 1
    

    unique_chars = [char for char in char_count if char_count[char] == 1]
    unique_chars.sort()
    return ''.join(unique_chars)

File: Java/프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.java
import java.util.*;

public class Solution {
    public String solution(String s) {
        HashMap<Character, Integer> charCountMap = new HashMap<>();
        for (char c : s.toCharArray()) {
            charCountMap.put(c, charCountMap.getOrDefault(c, 0) + 1);
        }
        List<Character> uniqueChars = new ArrayList<>();
        for (Map.Entry<Character, Integer> entry : charCountMap.entrySet()) {
            if (entry.getValue() == 1) {
                uniqueChars.add(entry.getKey());
            }
        }
        Collections.sort(uniqueChars);
        StringBuilder result = new StringBuilder();
        for (char c : uniqueChars) {
            result.append(c);
        }

        return result.toString();
    }
}